### PR TITLE
Add tags support to GraphiteBridge

### PR DESCRIPTION
**Pull request by partcyborg**
Graphite has support for tagged metrics with a syntax very similar to
the non-tagged format.  Update GraphiteBridge to support it.

Graphite's carbon tags format is explained here: https://grafana.com/blog/2018/01/11/graphite-1.1-teaching-an-old-dog-new-tricks

@brian-brazil

### CODE REVIEW

**Comment by partcyborg** [comment]
@csmarchbanks (new project maintainer)

**Review by csmarchbanks — COMMENTED** [review]
Thank you for your contribution!

A couple of small comments, but otherwise this looks good to me.

File: README.md [hunk 1]
```diff
@@ -440,13 +440,24 @@ Metrics are pushed over TCP in the Graphite plaintext format.
 ```python
 from prometheus_client.bridge.graphite import GraphiteBridge
 
-gb = GraphiteBridge(('graphite.your.org', 2003))
+gb = GraphiteBridge('graphite.your.org', 2003)
```
[the remark this answers is not in the document]
Comment (partcyborg) [reply README.md]: Oops. Fixed.

File: prometheus_client/bridge/graphite.py [hunk 2]
```diff
@@ -46,9 +46,10 @@ def run(self):
 
 
 class GraphiteBridge(object):
-    def __init__(self, address, registry=REGISTRY, timeout_seconds=30, _timer=time.time):
+    def __init__(self, address, registry=REGISTRY, tags=False, timeout_seconds=30, _timer=time.time):
```
[the remark this answers is not in the document]
Comment (partcyborg) [reply prometheus_client/bridge/graphite.py]: Moved tags to the very end

File: README.md [hunk 3]
```diff
@@ -440,13 +440,24 @@ Metrics are pushed over TCP in the Graphite plaintext format.
...
 # Push every 10 seconds in a daemon thread.
 gb.start(10.0)
 ```
 
+Graphite [tags](https://grafana.com/blog/2018/01/11/graphite-1.1-teaching-an-old-dog-new-tricks/) are also supported.
+
+```python
+from prometheus_client.bridge.graphite import GraphiteBridge
+
+gb = GraphiteBridge('graphite.your.org', 2003, tags=True)
```
[the remark this answers is not in the document]
Comment (partcyborg) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -447,6 +447,17 @@ gb.push()
 gb.start(10.0)
 ```
 
+Graphite [tags](https://grafana.com/blog/2018/01/11/graphite-1.1-teaching-an-old-dog-new-tricks/) are also supported.
+
+```python
+from prometheus_client.bridge.graphite import GraphiteBridge
+
+gb = GraphiteBridge(('graphite.your.org', 2003, tags=True))
```
Comment (csmarchbanks) [inline README.md:455]: Too many things in the tuple here :)

It should be `GraphiteBridge(('graphite.your.org', 2003), tags=True)`

**Review by csmarchbanks — APPROVED** [review]
:+1: Looks good, thanks!

**Comment by partcyborg** [comment]
Thanks for the quick review!

Do you have a general idea of when a prometheus_client release with this change might make its way to pypi?  It would just be nice to know when I stop building this from the git repo in our relevant Dockerfiles.

**Comment by csmarchbanks** [comment]
Good question, my estimate is within a month or so. As you saw, I recently took over as maintainer of this repo and am still on-boarding some.

**Comment by partcyborg** [comment]
Sounds good.  Thanks again and good luck with maintainership.

**Pull request merged**